contexts/MembershipContext/src/DataAccess/DoctrineApplicationRepository.php

```diff
@@ -142,13 +169,17 @@ private function getDoctrineStatus( Application $application ): int {
 			$status += DoctrineApplication::STATUS_CANCELED;
 		}
 
-		if ( $status === DoctrineApplication::STATUS_NEUTRAL ) {
+		if ( $status === DoctrineApplication::STATUS_NEUTRAL && $this->isDirectDebitPayment( $application ) ) {
 			$status += DoctrineApplication::STATUS_CONFIRMED;
```

Comment:
So this means that direct debit payments are "autoconfirmed"? Isn't there a confirmation that has to be set in the backend?